Makefile

```diff
@@ -64,6 +67,10 @@ lint: generated
 	gofumpt -l -w .
 	gci write . --skip-generated --skip-vendor
 	golangci-lint run
```

Comment:
Hmmm... We have duplication between `make` and `.github/workflows`. It would be nice to get rid of it so we don't have to think about supporting changes in two places.